Source/SIMPLib/Geometry/IGeometry.h

```diff
@@ -327,38 +327,38 @@ class SIMPLib_EXPORT IGeometry : public Observable
      * @brief getName
      * @return
      */
-    virtual QString getName() final;
+    virtual QString getName() const final;
```

Comment:
I know you are not the one who put it in, but can we just comment on how absurd `virtual final` is?  That seems to be happening a lot in this class.